haskell-open-source.md

```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
 ![](./pvp-chart.png)
 
 With `A=EPOCH`, `B=MAJOR`, and `C=MINOR`.
+
+## Release
+
+All (interested) Frecklers should be in the Maintainers group of our Hackage
+libraries, with permission to upload new versions.
+
+To release a new version:
+
+1. Update the package version
+1. Add a `CHANGELOG` entry
```

Comment:
Enumerate beyond 1?

Reply:
Bruh lrn2markdown :trollface: 

(Markdown automatically enumerates these for you when it renders.) https://github.com/freckle/guides/blob/dc92bdabeea48476df79f4bd47c076e94f954051/haskell-open-source.md

Reply:
Wait, do you not know about the feature of Markdown where you don't need to actually number lists?

![](https://files.pbrisbin.com/screenshots/screenshot.3801079.png)

Reply:
Wow feelin' pretty Markdown-shamed right now. Can't believe I've been typing out numbers all these years like some plebe. At least give me the moral victory of admitting this is some wack-looking syntax.

Reply:
https://xkcd.com/1053/

Reply:
This is my "least-favorite favorite" feature of markdown I was quite confused when I saw it for the first time, said "oh that's handy", and I'm still confused to this day.